deps/v8/.github/mistaken-pull-closer.yml

```diff
@@ -1,13 +0,0 @@
-# `true` will close all PRs.
```

Comment:
@targos Will this work with v8 sync script ?

Reply:
No, it won't. The V8 sync script bypasses our `.gitignore`.

Reply:
Reverted.